child-process-worker.js

```diff
@@ -2,14 +2,19 @@
 'use strict'
 
 const childProcess = require('child_process')
-const util = require('./util')
+const fs = require('fs')
+const path = require('path')
 
-const WORKER_PATH = '/tmp/worker.js' // path.join(__dirname, 'worker.js')
+const WORKER_PATH = '/tmp/fake-api-gateway-lambda/worker.js'
```

Comment:
Should we eventually use `os.tmpdir()` or https://github.com/sindresorhus/temp-dir ?

Reply:
:+1: willfix.